src/inbox/mod_inbox.erl

```diff
@@ -148,10 +155,27 @@ async_config_spec() ->
       }.
 
 process_inbox_boxes(Config = #{boxes := Boxes}) ->
-    false = lists:any(fun(Name) -> Name =:= <<"inbox">> orelse Name =:= <<"archive">> end, Boxes),
-    AllBoxes = [<<"inbox">>, <<"archive">> | Boxes ],
+    false = lists:any(fun(<<"inbox">>) -> true;
```

Comment:
I am not sure if naming a box `all` should be allowed due to its special meaning.